packages/vscode-extension/src/webview/providers/LaunchConfigProvider.tsx

```diff
@@ -18,11 +19,16 @@ const launchConfig = makeProxy<LaunchConfig>("LaunchConfig");
 type LaunchConfigContextType = LaunchConfigurationOptions & {
   update: LaunchConfigUpdater;
   xcodeSchemes: string[];
+  eas?: {
```

Comment:
is eas optional given it is set as empty object in default context?

Reply:
thx i removed it from default context